README.md

```diff
@@ -1,17 +1,21 @@
 # PuppeteerPdf
 
-This is a wrapper to NodeJS module [puppeteer_pdf](https://www.npmjs.com/package/puppeteer-pdf). After some attempts to use wkhtmltopdf using [pdf_generator](https://github.com/gutschilla/elixir-pdf-generator), I've decided to use other software to generate PDF and create a wrapper for it.
+This is a wrapper to NodeJS module [puppeteer-pdf](https://www.npmjs.com/package/puppeteer-pdf). After some attempts to use wkhtmltopdf using [pdf_generator](https://github.com/gutschilla/elixir-pdf-generator), I've decided to use other software to generate PDF and create a wrapper for it.
 
 ## Puppeteer PDF vs wkhtmltopdf
 
+I've written a [small blog post](https://medium.com/coletiv-stories/puppeteer-vs-wkhtmltopdf-and-why-i-created-a-new-module-9466eb1db7d1) where I explain my reasons to create this extension. Here is the list of pros and cons compared with `pdf_generator` module.
```

Comment:
we should link to the coletiv blog: https://coletiv.com/blog/elixir-pdf-generation-puppeteer-wkhtmltopdf/